src/block/carousel/save.js

```diff
@@ -77,31 +77,33 @@ export const Save = props => {
 			<BlockStyles.Content version={ props.version } attributes={ attributes } />
 			<CustomCSS.Content attributes={ attributes } />
 			<Separator.Content attributes={ attributes }>
-				<div className={ contentClassNames }>
-					<div
-						className="stk-block-carousel__slider"
-						role="list"
-						data-autoplay={ attributes.autoplay ? ( attributes.autoplaySpeed || '4000' ) : undefined }
-						data-label-slide-of={ attributes.ariaLabelPrev || 'Slide %%d of %%d' }
-					>
-						<InnerBlocks.Content />
-					</div>
-					{ attributes.showArrows && (
-						<div className="stk-block-carousel__buttons">
-							<button className="stk-block-carousel__button stk-block-carousel__button__prev" aria-label={ attributes.ariaLabelPrev || 'Previous slide' }>
-								<SvgIcon.Content
-									value={ attributes.arrowIconPrev || defaultIconPrev }
-								/>
-							</button>
-							<button className="stk-block-carousel__button stk-block-carousel__button__next" aria-label={ attributes.ariaLabelNext || 'Next slide' }>
-								<SvgIcon.Content
-									value={ attributes.arrowIconNext || defaultIconNext }
-								/>
-							</button>
+				<div className="stk-block-carousel__content-wrapper">
+					<div className={ contentClassNames }>
+						<div
+							className="stk-block-carousel__slider"
+							role="list"
+							data-autoplay={ attributes.autoplay ? ( attributes.autoplaySpeed || '4000' ) : undefined }
+							data-label-slide-of={ attributes.ariaLabelPrev || 'Slide %%d of %%d' }
+						>
+							<InnerBlocks.Content />
 						</div>
-					) }
+						{ attributes.showArrows && (
+							<div className="stk-block-carousel__buttons">
+								<button className="stk-block-carousel__button stk-block-carousel__button__prev" aria-label={ attributes.ariaLabelPrev || 'Previous slide' }>
+									<SvgIcon.Content
+										value={ attributes.arrowIconPrev || defaultIconPrev }
+									/>
+								</button>
+								<button className="stk-block-carousel__button stk-block-carousel__button__next" aria-label={ attributes.ariaLabelNext || 'Next slide' }>
+									<SvgIcon.Content
+										value={ attributes.arrowIconNext || defaultIconNext }
+									/>
+								</button>
+							</div>
+						) }
+					</div>
+					{ attributes.showDots && <div className="stk-block-carousel__dots" role="list" data-label={ attributes.ariaLabelSlide || 'Slide %%d' }></div> }
```

Comment:
Can you check if the different combinations of layout/locations for the dots and arrows work because the DOM arrangement of the dots and arrows were specifically made this way.